src/util.jl

```diff
@@ -28,6 +28,9 @@ end
     end
 end
 
+
+# TODO: the below seems to be type piracy...
+#=
```

Comment:
Personally, I'd lean toward just deleting any type piracy, and documenting it in the release notes.